lib/ios/RNNInterpolator.m

```diff
@@ -28,13 +28,39 @@ static CGFloat RNNApplyInterpolation(CGFloat p, RNNInterpolationOptions interpol
             return RNNLinear(p);
         case RNNInterpolationDecelerate:
             return RNNDecelerate(p);
+        case RNNInterpolationSpring:
+			// TODO: Expose springiness and mass properties to JS-API and make uniform with Android implementation (only has tension property)
+            return RNNSpring(p, 0.3, 3);
     }
 }
 
 static CGFloat RNNInterpolate(CGFloat from, CGFloat to, CGFloat p, RNNInterpolationOptions interpolation) {
     return from + RNNApplyInterpolation(p, interpolation) * (to - from);
 }
 
+static CGFloat RNNSpring(CGFloat p, CGFloat springiness, CGFloat mass) {
+	// TODO: Cache those allocations
```

Comment:
Why do we need to cache these variables?

Reply:
Those functions are being executed many times in a short amount of time, that's why my gut was telling me to remove everything that doesn't _have_ to be in there to somewhere else, especially "statics" like those variables to improve performance. They are being created with the values that the user supplies (calculated from `mass` and `springiness`) and won't change from there on forward.

Reply:
tbh it's just a bit of easy math, some `sqrt`s and `pow`s, nothing fancy like `cos`ines so we're probably fine and maybe the compiler will do some tricks anyways.